packages/core/src/utils.ts

```diff
@@ -0,0 +1,237 @@
+import { ExplorerUrl, IndexerUrl } from './constants.js';
+import { getFeeHandlerAddressesOfRoutes, getFeeHandlerTypeOfRoutes } from './multicall.js';
+import type {
+  Route,
+  TransferStatus,
+  TransferStatusResponse,
+  RouteIndexerType,
+  EnvironmentMetadata,
+  Domain,
+  RouteType,
+  SygmaConfig,
+  Domainlike,
+  Eip1193Provider,
+  FeeHandlerType,
+} from './types.js';
+import { Network, Environment } from './types.js';
+
+import { Config } from './index.js';
+
+function getIndexerTransferUrl(
+  env: Environment = process.env.SYGMA_ENV,
```

Comment:
We need to cast a type here otherwise it will be a string type 
```ts
env: Environment = process.env.SYGMA_ENV as Environment
```

Reply:
@Lykhoyda this type is defined in `environment.d.ts`

Reply:
https://github.com/sygmaprotocol/sygma-sdk/blob/b582bcdf922f72d43c3f6a12e20b9e3025e366de/packages/core/src/environment.d.ts

Reply:
It didn't work for the substrate, so I added this file to substrate package as well